src/Main.elm

```diff
@@ -25,42 +90,118 @@ init =
+                    let
+                        ( updatedModel, cmd, updatedTags ) =
+                            Tags.update (UserGroup.getTags userGroup) sectionMsg sectionModel
+                    in
+                    ( Success
+                        { successData
+                            | section = TagsSection updatedModel
+                            , userGroup = UserGroup.updateTags updatedTags userGroup
+                        }
+                    , Cmd.map TagsMsg cmd
+                    )
+
+                _ ->
+                    ( model
+                    , Cmd.none
+                    )
+
+        Error _ ->
+            ( model
+            , Cmd.none
+            )
 
 
 
 ---- VIEW ----
 
 
-header : String -> Html msg
-header text =
-    Html.span [ Attrs.class "p-2 text-5xl font-extrabold text-transparent bg-clip-text bg-gradient-to-br from-slate-400 to-slate-800" ]
-        [ Html.text text ]
-
-
-subheader : String -> Html msg
-subheader text =
-    Html.span [ Attrs.class "p-2 text-2xl font-extrabold text-slate-800" ]
-        [ Html.text text ]
-
-
 view : Model -> Html Msg
-view _ =
-    Html.div [ Attrs.class "flex flex-col w-[1024px] items-center mx-auto mt-16 mb-48" ]
-        [ header "Let's start your task"
-        , subheader "Here are your data:"
-        , Html.pre [ Attrs.class "my-8 py-4 px-12 text-sm bg-slate-100 font-mono shadow rounded" ] [ Html.text Data.userGroup ]
-        , header "Now turn them into form."
-        , subheader "See README for details of the task. Good luck 🍀 "
+view model =
+    case model of
+        Success ({ section } as successModelData) ->
+            Html.div [ Attributes.class "flex flex-col w-[1024px] items-center mx-auto mt-16 mb-48" ]
+                [ section
+                    |> sectionToTab
+                    |> Tabs.view
+                    |> Html.map TabsMsg
+                , formView successModelData
+                ]
+
+        Error (DecoderError _) ->
+            Html.div [ Attributes.class "flex flex-col w-[1024px] items-center mx-auto mt-16 mb-48" ]
+                [ Html.text "Decoder error - omething went wrong ..."
+                ]
+
+
+formView : SuccessModelData -> Html Msg
+formView { section, userGroup } =
+    Html.div [ Attributes.class "p-4 rounded-lg bg-gray-50 dark:bg-gray-800" ]
+        [ case section of
+            ContactDetailsSection contactDetailsModel ->
+                ContactDetails.view (UserGroup.getContactDetails userGroup) contactDetailsModel
+                    |> Html.map ContactDetailsMsg
+
+            SettingsSection settingsModel ->
+                Settings.view (UserGroup.getSettings userGroup) settingsModel
+                    |> Html.map SettingsMsg
+
+            TagsSection tagsModel ->
+                Tags.view (UserGroup.getTags userGroup) tagsModel
+                    |> Html.map TagsMsg
         ]
 
 
 
----- PROGRAM ----
+---- ROGRAM ----
```

Comment:
? =)

Reply:
Hups :-D Not sure how this change happened - most probably accidentally and I was not checking these comments any more.